src/java.base/share/classes/com/sun/crypto/provider/AESCrypt.java

```diff
@@ -602,13 +602,6 @@ private void implDecryptBlock(byte[] in, int inOffset,
      * @exception InvalidKeyException  If the key is invalid.
      */
     private void makeSessionKey(byte[] k) throws InvalidKeyException {
```

Comment:
Can it still throw InvalidKeyException?

Reply:
Good catch.  The throws is no longer needed.